src/ui/public/dialogues/__tests__/safe_confirm.js

```diff
@@ -45,35 +47,45 @@ describe('ui/safe_confirm', function () {
   });
 
   context('after timeout completes', function () {
-    it('$window.confirm is invoked with message', function () {
+    it('confirmation dialog is loaded to dom with message', function () {
       $timeout.flush();
-      expect($window.confirm.calledWith(message)).to.be(true);
+      const confirmationDialogElement = document.querySelector('.confirm-dialogue');
```

Comment:
Instead of using classes we should use a `data-test-subj="confirmDialogue"` attribute selector. This uncouples our tests from our CSS.